components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2241,6 +2241,34 @@ void RewardsServiceImpl::SaveRecurringTip(
                      AsWeakPtr()));
 }
 
+void RewardsServiceImpl::OnTwitterPublisherInfoSaved(
+    SaveMediaInfoCallback callback,
+    int result,
+    ledger::PublisherInfoPtr publisher) {
+  if (Connected()) {
```

Comment:
should we return null and `LEDGER_ERROR` if `Connected()` is false so that caller knows about it? Right now we callback only if everything is ok

Reply:
Sure, will do.